doc/releases/changelog-0.8.1.md

```diff
@@ -0,0 +1,15 @@
+# Release 0.8.1
+
+<h3>New features</h3>
+
+<h3>Improvements</h3>
+
+<h3>Breaking changes</h3>
+
+<h3>Deprecations</h3>
+
+<h3>Bug fixes</h3>
+
+<h3>Contributors</h3>
+
+This release contains contributions from (in alphabetical order):
```

Comment:
Hey @rauletorresc, I'm a bit confused. Shouldn't this contain the ZNE entries that we are planning for 0.8.1, instead of being empty? (cc @rmoyard)

Reply:
You are right, I just added it as a template for convenience. Once all the commits are cherry-picked I will update the entries.

Reply:
I downgraded the PR to a draft, for consistency :)